packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -410,6 +410,14 @@ export class SwapQuoter {
         return whitelistedApiKeys.includes(apiKey);
     }
 
+    private _isTxOriginBlacklisted(txOrigin: string | undefined): boolean {
+        if (!txOrigin) {
+            return false;
+        }
+        const blacklistedApiKeys = this._rfqtOptions ? this._rfqtOptions.txOriginBlacklist : [];
```

Comment:
should this be a set?

Reply:
Yes, or a `Set` or a HashSet (or object, in JS)